apollo-router/src/plugins/traffic_shaping/cache.rs

```diff
@@ -270,11 +302,15 @@ fn filter_representations(
     }
 
     for (ty, (hit, miss)) in cache_hit {
-        tracing::event!(
-            Level::INFO,
+        tracing::info!(
```

Comment:
Are hit and miss the total number of actual hits and misses?
If so this should be a gauge.

Also please rename the metric to `histogram.apollo.router.operations.entity.cache` with a hard coded attribute `hit=true` and `hit=false`

Reply:
No it's a counter